server.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path');
 const express = require('express');
-const config = require('./config');
+const config = require('./.config');
```

Comment:
Ash , did you create a folder for config ? If so, is it just in the root directory?

Reply:
I did not create any folders for config. I thought you created just a file in the root directory. Did you create any folders?

Reply:
no, but the change of route  from . /config   fo /./config is throwing errors for me.

Reply:
Was the setup of config.js changed at all ? apologies if I'm missing something

Reply:
@Jhaines1988 I got it. Conventionally, the hidden/secret files are started with a dot(.) So, you can just rename your **config** to **_.config_**. It will work then.